src/main/java/net/rptools/maptool/client/ui/token/dialog/edit/TokenPropertiesDialog.form

```diff
@@ -1122,7 +1122,7 @@
                         <properties>
                           <actionCommand value="Find"/>
                           <name value="textStatblockSearchButton"/>
-                          <text resource-bundle="net/rptools/maptool/language/i18n" key="EditTokenDialog.action.hero.statBlockRTextScroll"/>
+                          <text resource-bundle="net/rptools/maptool/language/i18n" key="EditTokenDialog.action.hero.statBlockRTextScroll.label"/>
```

Comment:
Forgot to change `statBlockRTextScroll` to `statBlockSearch` :slightly_smiling_face:

Reply:
oops, should be fixed now